app/views/admin/beer_styles/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.array!(@beer_styles) do |beer_style|
+  json.extract! beer_style, :id, :name, :full_bodied_ratio, :bitter_taste_ratio, :aroma_ratio
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)